README.md

```diff
@@ -17,28 +22,64 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# **KubeTurbo** <sup><sub>_Autonomic Scheduling for Kubernetes Implementations with Turbonomic_</sub></sup>
+# **KubeTurbo** 
 
 ----
 
 1. [Overview](#overview)
 2. [Getting Started](#getting-started)
-  * [System Requirements](#system-requirements)
+3. [Kubeturbo Use Cases](#use-cases)
+4. [Coming Soon](#coming-soon)
 
 ----
 
 ## Overview 
 
-Turbonomic will discover application containers 
-running across all pods in a Kubernetes cluster. In order to add this target, you must create a Kubeturbo configuration file which includes your Turbonomic credentials,and a custom pod definition that will be used by Kubelet to create a mirror pod running the Kubeturbo service.
+Kubeturbo leverages [Turbonomic](https://turbonomic.com/) patented analysis engine to provide visibility and control across the entire stack, to assure the performance of running micro-service in Kubernetes Pods, as well as the efficiency of underlying infrastructure.
```

Comment:
Question: Should that say "micro-service**s**?" I'm not familiar enough with the nomenclature to say for sure...